pkg/kubelet/cm/topologymanager/fake_topology_manager.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package topologymanager
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
This is out of scope of this PR it seems. Please, remove.

Reply:
Done.